http-server/src/main/java/com/facebook/airlift/http/server/HttpServer.java

```diff
@@ -159,6 +162,9 @@ public HttpServer(HttpServerInfo httpServerInfo,
             this.requestLog = null;
         }
 
+        int schedulerConcurrency = config.getSchedulerConcurrency();
+        int threadsPerScheduler = max(1, config.getSchedulerThreads() / schedulerConcurrency);
+        ConcurrentScheduler concurrentScheduler = new ConcurrentScheduler(schedulerConcurrency, threadsPerScheduler, "http-scheduler-%s");
```

Comment:
Maybe extract [this](https://github.com/prestodb/airlift/blob/master/http-client/src/main/java/com/facebook/airlift/http/client/jetty/JettyHttpClient.java#L434) method into some utility class (can be a static utility method in the ConcurrentScheduler) and reuse it here